MESH-3583 : [Syneos/GSK_B_Sure_gsk206882] - iOS - Align the step description text with the button labels (es_ES)

ricardobraverman opened the pull request:
Update the translation of the "DONE" button, but only in completion steps and only if said new translation is available.

Commit:
MESH-3583

[Syneos/GSK_B_Sure_gsk206882] - iOS - Align the step description text with the button labels (es_ES)

lopsae commented on ResearchKit/Localized/es.lproj/ResearchKit.strings:
```diff
@@ -204,6 +204,7 @@
 "BUTTON_APPLY" = "Aplicar";
 "BUTTON_DISAGREE" = "No acepto";
 "BUTTON_DONE" = "OK";
+"COMPLETION_BUTTON_DONE" = "LISTO";
```
Linguistic nitpick: none of the other button actions is all-caps (except `OK` because that one is an odd case). I feel this would be `Listo`, otherwise it will look too prominent compared to other actions.

ricardobraverman replied:
@lopsae I have the same feeling, but if you take a look at the ticket, we need to have this match with the completion step's text, which in turn will be fitted to pair with the SAW implementation for the button, which says LISTO.

ricardobraverman replied:
@lopsae Anyway, I'm going to doublecheck this with the TPM once more. I think we should go with capitalized case here and accept the discrepancy vs SAW, which renders the button's labels in caps.

ricardobraverman replied:
@lopsae Just FYI, I've pushed for a change and was allowed to set it to "Listo" instead.

Commit:
MESH-3583

 [Syneos/GSK_B_Sure_gsk206882] - iOS - Align the step description text with the button labels (es_ES)

lopsae reviewed (approved):
Woots less caps!
